.gitignore

```diff
@@ -8,7 +8,6 @@
 *.user
 *.userosscache
 *.sln.docstates
-.vscode/
```

Comment:
I would consider keeping this line and moving all VSCode config to `.vscode.example` so that the developer could tune the configuration to their own preferences.

But we can also consider removing them. See a previous comment by  @reyang (@bruno-garcia had a similar one): https://github.com/open-telemetry/opentelemetry-dotnet/pull/2015#issuecomment-828624988

Reply:
The `CONTRIBUTING.md` doc specifically lists VS Code as a supported way to work on this project. The whole point of providing the `.vscode` folder is to allow that to happen. [The opinion on this has changed since that discussion in #2015.](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2969#issuecomment-1062145699)